src/loggers/GoogleCloudLogger.js

```diff
@@ -54,7 +54,9 @@ export default class GoogleCloudLogger extends BaseLogger {
     this.emitPayload({
       message,
       severity,
-      ...this.getAdditionalFields(info),
+      userId: info.userId,
```

Comment:
how do you set the userId

Reply:
This already existed before (just moved around). It's captured from the `authUser` on the request